app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        new_person = Person(person.get("name"), person.get("age"))
+        person_list.append(new_person)
```

Comment:
List comprehension can be used effectively to create the person_list in a more concise and efficient manner.